### PR TITLE
Gives the Lawyer a pet goldfish

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

The law office now has a pet, a goldfish named McGill.

## Why It's Good For The Game

I've wanted to integrate aquariums and fishes in general into more roundstart, since it currently either requires being built and/or ordered, and thought giving it as a pet to the Law Office would be pretty cool since the law office doesn't have one and deserves one, and also just a silly reference.

## Changelog

:cl:
add: The Law office now has a pet goldfish.
/:cl:

### CODE REVIEW

**Comment by improvedname** [comment]
bob

**Comment by nfreader** [comment]
Saul is a perfect name for a lawyers goldfish.

**Comment by jughu** [comment]
now make it a spy objective to steal saul

**Comment by SomeoneYouProbablyKnow** [comment]
Give saul his own little telephone.

**Comment by DrAmazing343** [comment]
Make sure the Lawyer's offices start with fish food so Saul doesn't necessitate a roundstart SCRAMBLE for food!

**Comment by the-orange-cow** [comment]
Scales(as in the scales of justice, and fish scales)

File: code/modules/fishing/fish/fish_types.dm [hunk 1]
```diff
@@ -13,6 +13,11 @@
 	required_temperature_min = MIN_AQUARIUM_TEMP+18
 	required_temperature_max = MIN_AQUARIUM_TEMP+26
 
+/obj/item/fish/goldfish/saul
+	name = "Saul"
```
Comment (ATH1909) [inline code/modules/fishing/fish/fish_types.dm:17]: should he be given a custom description as well?

**Comment by kawoppi** [comment]
if you're referencing better call saul why not call it McGill (because it's a fish)

**Comment by JohnFulpWillard** [comment]
> Make sure the Lawyer's offices start with fish food so Saul doesn't necessitate a roundstart SCRAMBLE for food!

Their food should last like at least 90 mins but I didn't check anything to know exact numbers. I gave them a thing of fish food in their vendor though

**Comment by DrAmazing343** [comment]
> I gave them a thing of fish food in their vendor though

Peak, thank you.

**Review by Maurukas — APPROVED** [review]

**Pull request merged**